server/service/service.go

```diff
@@ -80,15 +81,27 @@ func New(conf *config.Config) (*Service, error) {
 			return nil, err
 		}
 
-		// ex) $HOME/config/data_encryption_file.sealed
-		// dir = $HOME/config/, file = data_encryption_file.sealed
-		dir, _ := filepath.Split(conf.DataEncryptionKeyFile)
-		err = tos.EnsureDir(dir, 0755)
+		var sealedSavedDir strings.Builder
+
+		userHomeDir, err := os.UserHomeDir()
+		if err != nil {
+			return nil, err
+		}
+
+		dir, file := filepath.Split(conf.DataEncryptionKeyFile)
+
+		// ex) .dataval/config/data_encryption_file.sealed
+		// sealedSavedDir = $HOME/.dataval/config/, file = data_encryption_file.sealed
+		sealedSavedDir.WriteString(userHomeDir)
+		sealedSavedDir.WriteString("/")
+		sealedSavedDir.WriteString(dir)
+
+		err = tos.EnsureDir(sealedSavedDir.String(), 0755)
```

Comment:
As I commented, the `data_encryption_key.sealed` file will be saved in `$HOME/.dataval/config` directory.